### PR TITLE
[#146]  adds sentry tags

### LINKED ISSUE #146: Adding sentry tags via logger properties
I would like a way to add tags to my sentry events via `_logger.Log` generic logger.
I don't want to have to call Sentry specific functions to add tags, the abstract logger provided by Microsoft can already provide these "properties" or "structured logging" to Sentry, so sentry should be able to pick them up and add them as tags.

### PR DESCRIPTION
**Pull request by odannyc**
Adds sentry tags based on logger props (state)

fixes #146

### CODE REVIEW

**Comment by bruno-garcia** [comment]
LGTM! Thanks @odannyc

File: src/Sentry.Extensions.Logging/SentryLogger.cs [hunk 1]
```diff
@@ -78,6 +78,11 @@ public bool IsEnabled(LogLevel logLevel)
                             };
                             break;
```
Comment (untitaker) [inline src/Sentry.Extensions.Logging/SentryLogger.cs]: Won't this line prevent the code below from running?

**Comment by untitaker** [comment]
FWIW in Python we add this stuff to extra instead of tags.

File: src/Sentry.Extensions.Logging/SentryLogger.cs (hunk 1 above)
Comment (bruno-garcia) [reply src/Sentry.Extensions.Logging/SentryLogger.cs]: That's true. Made sense before we added the new block but now if the `{OriginalFormat}` comes in first, the rest won't be added.
I'll fix it before publishing.

**Comment by bruno-garcia** [comment]
@untitaker replaced `break;` with `continue;`.
Discussed offline the reason for the tags there. 
i.e: `logger.Log("A request to {path} by user {userId}", path, userId)` will result in tags with the key from the template and value as the arg. LogEntry interface still used do sentry displays interpolated message but tags are indexed to be searchable.

**Pull request merged**